projects/gui/src/settingsdlg.cpp

```diff
@@ -95,6 +111,42 @@ void SettingsDialog::browseTbPath()
 	dlg->open();
 }
 
+void SettingsDialog::browseDefaultPgnFile()
```

Comment:
Changed to suggested behaviour. In between the directory had been set to the filename in the line edit, so the directory argument incorrectly was given the filename. The Qt selector figured it out nevertheless.